src/Dependencies/Installer.php

```diff
@@ -96,4 +87,22 @@ private function installDependency($type, $dep)
 
         return \Result\Result::ok();
     }
+
+    private function inspectionDetailsMessage($type, $dep)
+    {
+        $result = $this->inspectionChecker->check($type, $dep);
+
+        if (!$result->isErr()) {
+            $inspectionDetails = $result->unwrap();
+            if (!empty($inspectionDetails)) {
+                $message = sprintf("%s\n", $inspectionDetails);
```

Comment:
If you really want to avoid the nested conditionals, you could do something like:

```
$message = null;
if (!$result ->isErr() && !empty($result->unwrap())) {
    $message = sprintf("%s\n", $result->unwrap());
} else {
    $error = $result->getErr();
    $message = sprintf("[ERROR] %s\n", $error);
}
return $message;
```

But I'm not really convinced that's any better, so I'll leave it up to you.